package.json

```diff
@@ -17,7 +17,7 @@
   ],
   "scripts": {
     "circleci": "run-s test",
-    "css": "cssnano node_modules/normalize.css/normalize.css --safe > docs/dist/normalize.min.css",
```

Comment:
Is `--safe` flag needed yet? I couldn't find the description of `--safe` flag in documents of postcss & cssnano.

Reply:
Oh, I couldn't find it too.
`default presets` has included the `safe`.
https://cssnano.co/guides/optimisations

I'll fix it.

Reply:
e1198ab  fixed